exonum-java-binding/core/src/test/java/com/exonum/binding/core/runtime/TransactionMethodExtractorTest.java

```diff
@@ -115,6 +119,18 @@ void findMethodsValidServiceInterfaceImplementation() throws Exception {
     assertThat(actualMethods).containsExactlyInAnyOrderElementsOf(transactions.values());
   }
 
+  @Test
+  void findTransactionMethodsValidServiceProtobufArguments() throws Exception {
+    Map<Integer, Method> transactions =
+        TransactionMethodExtractor.findTransactionMethods(ValidServiceProtobufArgument.class);
+    assertThat(transactions).hasSize(1);
+    Method transactionMethod =
+        ValidServiceProtobufArgument.class.getMethod("transactionMethod",
+            TestProtoMessages.Point.class, TransactionContext.class);
+    assertThat(singletonList(transactionMethod))
```

Comment:
The assertion is the other way around: assertthat(actualValue).matchesSomeCondition(conditionParams)